pywbem_mock/_baseprovider.py

```diff
@@ -41,34 +41,22 @@
 from __future__ import absolute_import, print_function
 
 from pywbem import CIMError, \
-    CIM_ERR_NOT_FOUND, \
+    CIM_ERR_NOT_FOUND, CIMInstance, CIMClass, \
     CIM_ERR_ALREADY_EXISTS, CIM_ERR_INVALID_NAMESPACE, \
-    CIM_ERR_NAMESPACE_NOT_EMPTY
+    CIM_ERR_NAMESPACE_NOT_EMPTY, \
+    WBEMServer
+
+from pywbem._nocasedict import NocaseDict
 
 from pywbem._utils import _format
 
 # pywbem_mock implementation configuration variables that are used in
 # request responsders.
 
-# Default Max_Object_Count for Fake Server if not specified by request
-_DEFAULT_MAX_OBJECT_COUNT = 100
-
-# Maximum Open... timeout if not set by request
-OPEN_MAX_TIMEOUT = 40
-
 # Per DSP0200, the default behavior for EnumerateInstance DeepInheritance
 # if not set by server.  Default is True.
 DEFAULT_DEEP_INHERITANCE = True
```

Comment:
We now have the DEFAULT_DEEP_INHERITANCE parameter twice, once here and once in _mainprovider.py. Probably this copy here should be removed?

Reply:
DONE